main.py

```diff
@@ -1,10 +1,55 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    races = {}
+    skills = {}
+    guilds = {}
+
+    for player_name, player_data in data.items():
+        race_name = player_data["race"]["name"]
```

Comment:
use get() method instead of parenthesis for getting value by key to avoid KeyError